cli_proton_python/formatter.py

```diff
@@ -257,6 +258,15 @@ def print_message_as_interop(self):
 
         return int_result
 
+    def print_message_as_json(self):
+        """
+        Print message in JSON form
+
+        :return: message to be printed in JSON form
+        :rtype: str
+        """
+        return json.dumps(eval(self.print_message_as_interop()))
```

Comment:
Using eval is bad practise, use ast.literal_eval instead. With the use of ast we lose python2.4 compatibility, but I think it's reasonable to drop that support. Please don't forget to do the related update to README file.